overrides.go

```diff
@@ -0,0 +1,72 @@
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at http://mozilla.org/MPL/2.0/.
+//
+// Contributor:
+// - Brandon Myers bmyers@mozilla.com
+
+package main
+
+import (
+  "fmt"
+  "os"
+  "bufio"
+  "strings"
+  "strconv"
+)
+
+type Override struct {
+  CIDR string
+  City string
+  Country string
+  Latitude float64
+  Longitude float64
+}
+
+
+func readOverrides(path string) (overrides []Override){
+  file, err := os.Open(path)
+  defer file.Close()
+  if err != nil {
+    fmt.Printf("error opening file: %v\n",err)
```

Comment:
We should probably use fmt.Fprintf to os.Stderr here

Reply:
Sounds good to me, I've made that change in https://github.com/ameihm0912/geomodel/pull/10/commits/f6a43a5b9059ed118dc3455fdbb8b64fd1ef45fa